auth_password_settings/models/res_config.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Dhaval Patel
+#    Copyright (C) 2011 - TODAY Denero Team. (<http://www.deneroteam.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp.osv import (
+    osv,
+    fields)
+from openerp.tools.safe_eval import safe_eval
+from openerp.tools.translate import _
+
+
+class BaseConfigSettings(osv.osv_memory):
```

Comment:
`osv.osv_memory` is deprecated.  Please use 'orm.TransientModel'

Since this is a new module for v8, can you consider porting it to the new API?